src/app/games/quarto/QuartoRules.ts

```diff
@@ -128,7 +128,7 @@ class Critere {
     public isAllNull(): boolean {
         let i: number = 0;
         do {
-            if (this.subCritere[i] !== null) {
```

Comment:
faut voir si y'a une règle eslint pour pas faire acepter les !== null !

Reply:
y'a bien l'inverse (forcer la distinction entre null et undefined) mais pas cette règle :D
D'ailleurs, pourquoi mixe-t-on deux types ?

Reply:
Bah, car quand on test != null c'est pour filtrer en amont les null pointer, avec un !== bah, on filtre qu'un type de null pointer sur deux. C'qu'est pas intéréssant

Reply:
Est-ce qu'ici la condition devrait pas être du genre `if (i is a key in this.subCritere && this.subCritere[i] !== null)` ? Car ça semble confondre les deux le "!="